api/v1alpha1/kedacontroller_types.go

```diff
@@ -32,6 +32,7 @@ const (
 // KedaControllerSpec defines the desired state of KedaController
 // +kubebuilder:subresource:status
 type KedaControllerSpec struct {
+	// Important: Run "make" to regenerate code after modifying this file
```

Comment:
this is the reason why this comment is showing in the generated CRD (what I have mentioned below).

Pls replace this comment (and the same for the rest of the Status, you can take an inspiration from the sample or https://github.com/kedacore/keda/blob/main/BUILD.md#keda-operator-logging